pycc/local.py

```diff
@@ -418,6 +426,8 @@ def filter_res(self, r1, r2):
             j = ij % no
 
             X = self.Q[ij].T @ r2[i,j] @ self.Q[ij]
+            Y = self.L[ij].T @ X @ self.L[ij]
+            X = self.L[ij] @ Y @ self.L[ij].T
```

Comment:
Just to confirm, we need the L-transformation on the residuals to remove the linear dependencies of the LPAO basis only, right?  And for LPNO, the L-transformation is orthogonal?

Reply:
precisely - this transformation does nothing for the LPNO case. I could save the `local` string (currently the only parameter from `Local()` that isn't saved as a class variable) and do an `if` statement here, if you'd like.